Index: out/production/JAVA-Placement-Series/.idea/.gitignore
===================================================================
diff --git a/out/production/JAVA-Placement-Series/.idea/.gitignore b/out/production/JAVA-Placement-Series/.idea/.gitignore
deleted file mode 100644
--- a/out/production/JAVA-Placement-Series/.idea/.gitignore	(revision c2b9e1c6422b1570d6ba0796df2b4a6970aab9e5)
+++ /dev/null	(revision c2b9e1c6422b1570d6ba0796df2b4a6970aab9e5)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: CollectionInterface/AutoBoxingAndUnboxingDemo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CollectionInterface/AutoBoxingAndUnboxingDemo.java b/CollectionFramework/AutoBoxingAndUnboxingDemo.java
rename from CollectionInterface/AutoBoxingAndUnboxingDemo.java
rename to CollectionFramework/AutoBoxingAndUnboxingDemo.java
--- a/CollectionInterface/AutoBoxingAndUnboxingDemo.java	(revision c2b9e1c6422b1570d6ba0796df2b4a6970aab9e5)
+++ b/CollectionFramework/AutoBoxingAndUnboxingDemo.java	(date 1671954488128)
@@ -1,4 +1,4 @@
-package CollectionInterface;
+package CollectionFramework;
 
 public class AutoBoxingAndUnboxingDemo {
     public static void main(String[] args) {
